WhenWorkingWithAListOfPetNames.java

```diff
@@ -0,0 +1,94 @@
+package serenitylabs.tutorials.vetclinic.collections.exercises;
+
+import com.google.common.collect.Lists;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.hasItems;
+import static org.junit.Assert.assertThat;
+
+public class WhenWorkingWithAListOfPetNames {
+
+    @Test
+    public void should_add_Fido_to_the_list_of_pets() {
+        List<String> names = Lists.newArrayList();
+
+        names.add("Fido");
+
+        assertThat(names, hasItems("Fido"));
+    }
+
+    @Test
+    public void should_remove_Fido_from_the_list_of_pets() {
+        List<String> names = Lists.newArrayList("Felix","Fido","Spot");
+
+        names.remove("Fido");
+        assertThat(names, hasItems("Felix","Spot"));
+    }
+
+    @Test
+    public void should_remove_the_first_pet_from_the_list_of_pets() {
+        List<String> names = Lists.newArrayList("Felix","Fido","Spot");
+
+        names.remove(0);
+
+        assertThat(names, hasItems("Fido","Spot"));
+    }
+
+    @Test
+    public void should_make_a_list_of_cats_and_dogs() {
+        List<String> cats = Lists.newArrayList("Felix","Spot");
+        List<String> dogs = Lists.newArrayList("Fido","Rover");
+
+        List<String> catsAndDogs = null;
+        catsAndDogs = new ArrayList<String>();
```

Comment:
This is correct, but normally you would remove the null and just assign the list directly, e.g.
```
List<String> catsAndDogs = new ArrayList<>();
```